From 670fc1ae14eb4517cb971806a9608b22ef28ab39 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Wed, 6 Jul 2016 12:23:22 +0200
Subject: [PATCH 3/6] abuild: perform unzip operations quietly

unzip(1) is verbose by default since we don't run tar(1) in verbose mode
we should run unzip with -q for consistency.
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index 883c661..72f8978 100644
--- a/abuild.in
+++ b/abuild.in
@@ -410,7 +410,7 @@ default_unpack() {
 				unxz -c "$s" | tar -C "$srcdir" -x || return 1;;
 			*.zip)
 				msg "Unpacking $s..."
-				unzip "$s" -d "$srcdir" || return 1;;
+				unzip -q "$s" -d "$srcdir" || return 1;;
 		esac
 	done
 }
-- 
2.9.0

